object/organization.go

```diff
@@ -495,3 +506,32 @@ func (org *Organization) GetInitScore() (int, error) {
 		return strconv.Atoi(conf.GetConfigString("initScore"))
 	}
 }
+
+func updateOrganizationUsersPasswordChangeTime(owner string, oldInterval int, newInterval int) error {
+	if newInterval == 0 {
+		bean := make(map[string]interface{})
+		bean["password_change_time"] = nil
+		ormer.Engine.Table("user").Where("password_change_time >= now() and owner = ?", owner).Cols("password_change_time").Update(bean)
+	} else {
+		users := []*User{}
+		err := ormer.Engine.Table("user").Where("password_change_time >= now() or password_change_time is null and owner = ?", owner).Find(&users)
+		if err != nil {
+			return err
+		}
+		for _, user := range users {
+			if user.PasswordChangeTime.IsZero() {
+				user.PasswordChangeTime = getNextPasswordChangeTime(newInterval)
+			} else {
+				user.PasswordChangeTime = user.PasswordChangeTime.
+					Add(-1 * time.Hour * 24 * time.Duration(oldInterval)).
+					Add(time.Hour * 24 * time.Duration(newInterval))
+			}
+
+			_, err := ormer.Engine.ID(core.PK{user.Owner, user.Name}).Cols("password_change_time").Update(user)
```

Comment:
can we use tx?

Reply:
yep, now updateOrganization method entirely works in transaction